mkosi.md

```diff
@@ -674,7 +674,8 @@ details see the table below.
 
 `--autologin`
 
-: Enable autologin for the `root` user.
+: Enable autologin for the `root` user on pts/0 (nspawn), tty1 (QEMU) and
```

Comment:
Maybe "enable autologin by patching `/etc/pam.d/login`" to give users a pointer where to look if they want / need to change something about this and can decide whether this is the appropriate option for them.